migrations/20170413101210_flat.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('flat', function (table) {
+    table.increments('id').primary()
+    table.string('name')
+    table.integer('bills_id')

```

Comment:
you shouldn't need bills_id as the relationship is a flat has many bills, not the other way around